app/tests/search/test_dataone.py

```diff
@@ -305,8 +305,39 @@ def test_Bounding_box_geometry(self):
             'westBoundCoord': -105.5341,
             'eastBoundCoord': -105.5341
         }
+
         # Testing for a point
         test_obj_2 = self.search.convert_result(test_result2)
         self.assertEqual(test_obj_2.geometry.type, 'Point')
         self.assertEqual(test_obj_2.geometry.coordinates, (76.7067, -105.5341))
 
+
+    def test_original_datasource(self):
+
+        self.search.max_score = 10
```

Comment:
I think you don't need `max_score` anymore (did the test fail without it? if so, that's a bug)